Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/Pass.h

```diff
@@ -153,7 +153,10 @@ namespace AZ
             virtual bool IsEnabled() const { return m_flags.m_enabled; }
 
             bool HasDrawListTag() const { return m_flags.m_hasDrawListTag; }
-            bool HasPipelineViewTag() const { return m_flags.m_hasPipelineViewTag; }
+            bool BindViewSrg() const
+            {
+                return m_flags.m_bindViewSrg;
+            }
```

Comment:
This looks like a breaking public API change - I think we should keep the old function for deprecation purposes for one release - and increment the minor gem version in `Gems/Atom/RPI/gem.json` from `0.0.0` to `1.0.0`

```suggestion
            // Deprecated - replaced with BindViewSrg()
            bool HasPipelineViewTag() const { return BindViewSrg(); }
            bool BindViewSrg() const
            {
                return m_flags.m_bindViewSrg;
            }
```